examples/simple-reply/asyncapi-nats-client/README.md

```diff
@@ -1,7 +1,7 @@
 
 <h1 align="center">TypeScript/Node.js NATS client wrapper</h1>
 <p align="center">
-  <em>This is a generated TypeScript/Node.js NATS client for the application - Streetlight.</em>
+  <em>This is a generated TypeScript/Node.js NATS client for the application - Reply example with streetlight.</em>
```

Comment:
sorry, what? O_O

"Reply example with streetlight."

Reply:
Auto-generated markdown code based on https://github.com/asyncapi/ts-nats-template/blob/master/template/README.md.js

Depends on the provided AsyncAPI document, so it is dynamic 🙂